client/src/network.ts

```diff
@@ -45,11 +44,18 @@ export class NetworkHandler {
 
     public sendInput(input: LocalPlayerInput) {
         this.client.send(JSON.stringify({
-            frame: input.frame,
-            input: [].slice.call(input.input),
+            input: {
+                frame: input.frame,
+                input: [].slice.call(input.input),
```

Comment:
What is the purpose of slicing an empty array?

Reply:
To convert `Uint8Array` to `Array` - because `new Uint8Array([1, 2, 3, 4])` gets serialized as `{"0":1,"1":2,"2":3,"3":4}`